vag2wav/vag2wav.c

```diff
@@ -12,94 +12,94 @@
 #include "endian.c"
 
 double f[5][2] = { { 0.0, 0.0 },
-                    {   60.0 / 64.0,  0.0 },
-                    {  115.0 / 64.0, -52.0 / 64.0 },
-                    {   98.0 / 64.0, -55.0 / 64.0 },
-                    {  122.0 / 64.0, -60.0 / 64.0 } };
+					{   60.0 / 64.0,  0.0 },
+					{  115.0 / 64.0, -52.0 / 64.0 },
+					{   98.0 / 64.0, -55.0 / 64.0 },
+					{  122.0 / 64.0, -60.0 / 64.0 } };
```

Comment:
This is ugly as hell.
```suggestion
double f[5][2] = { { 0.0, 0.0 },
	{ 60.0 / 64.0, 0.0 },
	{ 115.0 / 64.0, -52.0 / 64.0 },
	{ 98.0 / 64.0, -55.0 / 64.0 },
	{ 122.0 / 64.0, -60.0 / 64.0 } };
```
I recommend using devilutionx's clang-format settings